### PR TITLE
Feature/ub bv 1111

### PR DESCRIPTION
**Pull request by ubozov**
These commits contains the following changes:
1. Starts services after install.
2. Stores env from `key-value` to `json`.
3. Change owner to resources.
4. Usage `exe` path.
5. Usage config to paths.
6. Usage `adapter.config.json`.

Resolves BV-1111.

### CODE REVIEW

File: inst/openvpn/path/path.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ const (
 	// DappVPNConfig file location
 	DappVPNConfig = `config/dappvpn.config.json`
 	// DappCtrlConfig file location
-	DappCtrlConfig = `../../../dappctrl/dappctrl.config.json`
+	DappCtrlConfig = `../../dappctrl/dappctrl.config.json`
```
Comment (furkhat) [inline inst/openvpn/path/path.go]: Why?

Comment (drew2a) [inline inst/openvpn/path/path.go]: From the build scripts point of view, would be better to have all this paths in a config file.

Comment (ubozov) [reply inst/openvpn/path/path.go]: I changed the logic of determining the absolute path 
https://github.com/Privatix/dapp-openvpn/commit/d3a0b4965942fbb62ba8f28c56b1f2b78ab947b7#diff-aa9887825c98e9cb9e1be2d1c88fc3ebL52-L53

Comment (ubozov) [reply inst/openvpn/path/path.go]: updated

Comment (ubozov) [reply inst/openvpn/path/path.go]: paths configurates through `config/path.config.json`

**Pull request merged**